mautic/mautic
# Minor UI updates

## Description
This PR makes a few UI changes:

1. The connecting lines and arrows in the campaign builder were made thinner and smaller. 
2. Changes the apps link color to be Mautic blue by default and darken on hover (versus bootstrap default to lighten the primary color with primary on hover)
3. Fixed issue where Segments and Contacts did not turn gold on active
4. Moved assets and forms to a new Components menu item

## Steps to test

Apply the PR, clear Symfony's cache, then check the above areas in the UI. Use dev mode since prod assets are not updated.
